uvm/servlets/admin/config/administration/MainController.js

```diff
@@ -98,8 +98,8 @@ Ext.define('Ung.config.administration.MainController', {
             vm.set({
                 adminSettings: result[0],
                 systemSettings: result[1],
-                skinsList: result[2],
-                skinSettings: result[3]
+                //skinsList: result[2],
```

Comment:
Let's just delete it, not comment it out.

Reply:
removed